### PR TITLE
[T4A3][T15-A3]Kenny Ng

### PR DESCRIPTION
**Pull request by kennyngdsc**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```
Comment (sooyj) [inline src/seedu/addressbook/data/person/Contact.java:3]: The superclass is "thinner" than most :-). Question for you: should "isValid()" be a superclass method? Why?

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -6,23 +6,20 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact {
```
Comment (sooyj) [inline src/seedu/addressbook/data/person/Address.java:9]: Correct way to create a subclass.

**Review by sooyj — COMMENTED** [review]
Reasonable attempt. A couple of methods were decided(?) not to pushed to superclass (e.g. isValid, equals).

File: src/seedu/addressbook/data/person/Contact.java (hunk 1 above)
Comment (kennyngdsc) [reply src/seedu/addressbook/data/person/Contact.java:3]: I think maybe we can make Contact an abstract class so the method isValid() can be implemented in Contact without returning any value.

**Pull request closed without merging**